docs/reference/commandline/service_update.md

```diff
@@ -171,9 +171,9 @@ $ docker service update --mount-rm /somewhere myservice
 myservice
 ```
 
-### Rolling back to the previous version of a service 
+### Rolling back to the previous version of a service
```

Comment:
Don't use gerunds (-ing words) in titles. How about "Roll back to the previous version of a service"? By the way, when you are using the verb form, it is "roll back" rather than "rollback".